src/client/eppo-client.ts

```diff
@@ -90,22 +65,16 @@ export interface IEppoClient {
     flagKey: string,
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     subjectAttributes?: Record<string, any>,
+    defaultValue?: number | null,
     assignmentHooks?: IAssignmentHooks,
   ): number | null;
 
-  getJSONStringAssignment(
```

Comment:
I figured no one would use this -- hence removing it. Let me know if you disagree.